pipeline/lab.py

```diff
@@ -0,0 +1,409 @@
+import datajoint as dj
+import numpy as np
+from . import get_schema_name
+
+schema = dj.schema(get_schema_name('lab'))
+
+
+@schema
+class Person(dj.Manual):
+    definition = """
+    username : varchar(24) 
+    ----
+    fullname : varchar(255)
+    """
+
+
+@schema
+class Rig(dj.Manual):
+    definition = """
+    rig             : varchar(24)
+    ---
+    room            : varchar(20) # example 2w.342
+    rig_description : varchar(1024) 
+    """
+
+
+# ---- Animal information ----
+
+@schema
+class AnimalStrain(dj.Lookup):
+    definition = """
+    animal_strain       : varchar(30)
+    """
+    contents = zip(['pl56', 'kj18'])
+
+
+@schema
+class AnimalSource(dj.Lookup):
+    definition = """
+    animal_source       : varchar(30)
+    """
+    contents = zip(['Jackson Labs', 'Allen Institute', 'Charles River', 'MMRRC', 'Taconic', 'Other'])
+
+
+@schema
+class ModifiedGene(dj.Manual):
+    definition = """
+    gene_modification   : varchar(60)
+    ---
+    gene_modification_description = ''         : varchar(256)
+    """
+
+
+@schema
+class Subject(dj.Manual):
+    definition = """
+    subject_id          : int   # institution 6 digit animal ID
```

Comment:
We often mix letters and numbers to identify subjects 
```suggestion
    subject_id          : varchar(10)  # 
```